.travis/add_pod_annotations.sh

```diff
@@ -20,7 +20,7 @@ if $DIR/build-condition.sh $TRAVIS_COMMIT_RANGE projects/; then
     echo -e "\npodAnnotations:" >> charts/fairspace/values.yaml
     echo -e "  fairspace:\n    commit: \"$COMMIT_ID\"\n" >> charts/fairspace/values.yaml
 
-    echo "Changes in $project_name project detected, fairspace pod will be restarted."
+    echo "Changes in $project_name project detected, metatree pod will be restarted."
```

Comment:
the pod name in kubernetes setup was not changed (it's still fairspace), so this shouldn't be changed